src/main/java/com/google/devtools/build/lib/remote/blobstore/http/AbstractHttpHandler.java

```diff
@@ -30,7 +31,9 @@
 import java.util.List;
 import java.util.Map;
 
-/** Common functionality shared by concrete classes. */
+/**
```

Comment:
please revert this to one line. it's a style guide that we use all around Bazel.

Reply:
noted